app/models/user.rb

```diff
@@ -38,4 +39,25 @@ class User < ActiveRecord::Base
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable, :confirmable
   has_many :articles
+
+
+  ROLES = %w[:admin :blogger :viewer]
```

Comment:
please remove multiple new lines, is this code auto-generated? documentation suggests only adding two lines here, include RoleModel and available roles. refer:
https://github.com/martinrehfeld/role_model